### PR TITLE
suggested improvements of release notes

### PR DESCRIPTION
**Pull request by markus2330**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

**Comment by markus2330** [comment]
@mpranj these are only suggestions, please feel free to ignore as much as you want. You might want to copy the file, and do `git add -p` to only add specific parts. It is not important if the ownership of the commit is preserved.

I can do one more review by reading the rendered website, this can also be done after the release tag.

**Comment by mpranj** [comment]
Thank you so much for the suggestions, I will take a look and pick things soon.

The release tag and tarballs are basically done. I am just testing locally to make sure the tarball is not broken. After testing I will push everything.

Regarding the rendered website: improvements can be done at any point now, even if the news are published. I just don't want to publish completely broken text or a tarball which will not compile.

File: doc/news/_preparation_next_release.md [hunk 1]
```diff
@@ -85,7 +90,7 @@ New maintainers are very much welcomed!
 
 ### SWIG/python2
 
-- Removed. _(Manuel Mausz)_
+- Removed python2 plugin, as python2 support ended. _(Manuel Mausz)_
```
Comment (mpranj) [inline doc/news/_preparation_next_release.md:93]: :+1:

Comment (mpranj) [inline doc/news/_preparation_next_release.md:93]:
```suggestion
- Removed python2 binding, as python2 support ended. _(Manuel Mausz)_
```

File: doc/news/_preparation_next_release.md [hunk 2]
```diff
@@ -167,18 +172,17 @@ you up to date with the multi-language support provided by Elektra.
 
 ### python2
 
-- Removed. _(Manuel Mausz)_
+- Removed python2 binding, as python2 support ended. _(Manuel Mausz)_
```
Comment (mpranj) [inline doc/news/_preparation_next_release.md:175]:
```suggestion
- Removed python2 plugin, as python2 support ended. _(Manuel Mausz)_
```

File: doc/news/_preparation_next_release.md [hunk 3]
```diff
@@ -292,6 +296,22 @@ plugins, bindings and tools are always up to date. Furthermore, we changed:
...
+- Elektrify LCDproc _(Klemens Böswirth)_ and _(Jakob Fischer)_
+- Packaging for popular Linux distributions _(Djordje Bulatovic)_
+- Improve 3-way merge. _(Dominic Jäger)_
+- Go bindings and improved Web-UI _(Raphael Gruber)_
+- TOML plugin as new default storage _(Jakob Fischer)_
+- Shell completion _(Ulrike Schäfer)_
+- Improve Elektra developer experience _(Hani Torabi)_
+- Ansible bindings _(Thomas Waser)_
+- Plugin interface improvements _(Vid Leskovar)_
+
```
Comment (mpranj) [inline doc/news/_preparation_next_release.md:314]: Thank you for adding this section!

**Pull request merged**